data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_equip_arms_armor.lst

```diff
@@ -123,28 +123,28 @@ Wushu Dart (5)				PROFICIENCY:WEAPON|Wushu Dart					TYPE:Weapon.Resizable.Easter
+Pistol															PROFICIENCY:WEAPON|Pistol								TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:1000	WT:4										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.138	SPROP:Misfire 1 (5 ft)
+Pistol (Coat)							OUTPUTNAME:Coat Pistol				PROFICIENCY:WEAPON|Pistol (Coat)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=1				COST:750	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
+Pistol (Dagger)							OUTPUTNAME:Dagger Pistol			PROFICIENCY:WEAPON|Pistol (Dagger)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.Light.Finesseable.Simple.Standard.Piercing.Slashing.Dagger.BladeLight	CONTAINS:1|FirearmBullet=1				COST:740	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
+Pistol (Double-Barreled)					OUTPUTNAME:Double-Barreled Pistol		PROFICIENCY:WEAPON|Pistol (Double-Barreled)					TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol																					CONTAINS:1|FirearmBullet=2				COST:1750	WT:5										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d8					EQMOD:STEEL													WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.)
+Pistol (Dragon)							OUTPUTNAME:Dragon Pistol			PROFICIENCY:WEAPON|Pistol (Dragon)							TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Scatter																			CONTAINS:1|FirearmBullet=1|FirearmPellet=1	COST:1000	WT:3										CRITMULT:x4	CRITRANGE:1							DAMAGE:1d6					EQMOD:FIREARM_SCATTER.STEEL										WIELD:OneHandedOnly	RANGE:20				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1-2 (5 ft.); Scatter 15' cone
+Pistol (Sword Cane)						OUTPUTNAME:Sword Cane Pistol			PROFICIENCY:WEAPON|Pistol (Sword Cane)						TYPE:Weapon.Resizable.Exotic.Ranged.Standard.Piercing.Bludgeoning.Container.Projectile.Firearm.Pistol.Double		ALTTYPE:Weapon.Resizable.Melee.OneHanded.Martial.Standard.Piercing.Sword					CONTAINS:1|FirearmBullet=1				COST:775	WT:1										CRITMULT:x3	CRITRANGE:1							DAMAGE:1d4					EQMOD:STEEL										ALTEQMOD:STEEL	WIELD:OneHandedOnly	RANGE:10				SIZE:M	SOURCEPAGE:p.139	SPROP:Misfire 1 (5 ft.)
```

Comment:
Which one? I didn't change the SPROP on the culverin. I copied the other existing misfire or scatter cone lines for any I changed.